README.md

```diff
@@ -38,6 +38,7 @@ make test
 | nomad\_datacenters | Nomad data centers | list(string) | ["dc1"] | no |
 | nomad\_namespace | [Enterprise] Nomad namespace | string | "default" | no |
 | local_docker_image | Switch for nomad job | bool | - | yes |
+| use\_canary | Uses canary deployment for Hive | bool | false | no |
 | hive_service_name | Hive service name | string | "hive-metastore" | no |
```

Comment:
Why are some escaped and some are not?

Reply:
Why do we even have escaped underscores?

Reply:
Not to bring up old wounds, but did we get an answer to this? 😅